content/fair/interoperable/index.md

```diff
@@ -0,0 +1,141 @@
+
+### Describe the **scope**, and the users of the resource. How is Galaxy positioned with respect to other similar interoperability resources?
+
+The usegalaxy.* servers (in the US, EU, AUS) have an extensive user base each of tens of thousands of registered users, adding a few thousand new users every month. These users come from different scientific fields and are majorly non-technical scientists, who intend to answer research questions using data science methods. The majority of users is interested in analysing own data in the context of publicly available reference resources and data collections. Their main goal is to develop workflows of tools and to apply them on a large scale. According to this user profile, the Galaxy Training Network (GTN, https://training.galaxyproject.org) has developed broad and interactive high-quality training materials, being deeply rooted in both Galaxy’s community and technical infrastructure.
+
+Usegalaxy.eu has been developed as a European Galaxy project by multiple ELIXIR partners as well as individuals from all over Europe. It is positioned as a flagship project of the German Network of Bioinformatics infrastructure (de.NBI) and part of multiple national projects. It is one of the biggest ELIXIR services and the preferred gateway for scientific computing of multiple European Open Science Cloud (EOSC) projects. Usegalaxy.eu focuses on providing a platform for analysing and sharing scientific data rather than developing standards. In other words, the European Galaxy project progressively integrates concepts, standardisations, tools, APIs and concepts developed by related initiatives in order to bring them to application in users’ analyses and increase the overall level of practised interoperability.
+
+Base URLs (home pages): https://usegalaxy.org, https://usegalaxy.eu, https://usegalaxy.org.au
+
+Introductory page URL: https://galaxyproject.org
+
+### Does Galaxy have a community **recognition**? What about inter-organisational collaborations, geographical diversity in the source of the submissions, international diversity of delivery partners and/or funders?
+
+Recognition of Galaxy is broad and observed across multiple communities. Galaxy as a platform originates from the US (usegalaxy.org) and has been developed by a worldwide community since its initial release in 2005. [The Galaxy community](https://galaxyproject.org/community/) is traditionally open, welcoming, diverse and highly active. With Galaxy Europe and Galaxy Australia another two major instances have been founded, complemented with multiple national ones, several of them located in Europe.
```

Comment:
```suggestion
Recognition of Galaxy is broad and observed across multiple communities. Galaxy as a platform has been developed by a worldwide community since its initial release in 2005. [The Galaxy community](https://galaxyproject.org/community/) is open, welcoming, diverse, and highly active.
```

Reply:
And then maybe mention that there are 100s local Galaxy servers, national Galaxy server and 3 continental Galaxy servers. With links to the hub pages. I think this should be enough?